src/libpriv/rpmostree-output.c

```diff
@@ -0,0 +1,99 @@
+/*
+* Copyright (C) 2016 Red Hat, Inc.
+*
+* This program is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program; if not, write to the Free Software
+* Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+*/
+
+#include "config.h"
+
+#include <ostree.h>
+#include <libglnx.h>
+
+#include "rpmostree-output.h"
+
+/* These are helper functions that automatically determine whether data should
+ * be sent through an appropriate D-Bus signal or sent directly to the local
+ * terminal. This is helpful in situations in which code may be executed both
+ * from the daemon and daemon-less. */
+
+static GLnxConsoleRef console;
```

Comment:
Kind of weird to have this be static given libglnx already has a static version.  Wonder if we should have a ref-less API there.